nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -220,16 +220,14 @@ def _check_duplicates(image_series_kwargs_list):
 
                 # capture data in kwargs:
                 image_series_kwargs.update(data=data)
-                # capture time info in kwargs:
-                with VideoCaptureContext(str(file), stub=stub_test) as vc:
-                    timestamps = starting_times[j] + vc.get_movie_timestamps()
-                    if len(starting_times) != len(file_paths):
-                        starting_times.append(timestamps[-1])
-                    if check_regular_timestamps(ts=timestamps):
-                        fps = vc.get_movie_fps()
-                        image_series_kwargs.update(starting_time=starting_times[j], rate=fps)
-                    else:
-                        image_series_kwargs.update(timestamps=timestamps)
+                timestamps = starting_times[j] + video_capture_ob.get_movie_timestamps()
+                if len(starting_times) != len(file_paths):
+                    starting_times.append(timestamps[-1])
+                if check_regular_timestamps(ts=timestamps):
+                    fps = video_capture_ob.get_movie_fps()
+                    image_series_kwargs.update(starting_time=starting_times[j], rate=fps)
+                else:
+                    image_series_kwargs.update(timestamps=timestamps)
```

Comment:
what is wrong with the way you were doing it before?

Reply:
before, we were using `DataChunkIterator` for both `chunk_data=True/False`. 
When chunking is disabled then we should not use the `DataChunkIterator` at all and just wrap the array with HDF5IO to compress. What do you think @CodyCBakerPhD since you had set it up like this initially?

Additionally, in another PR we would be using the custom `MovieDataChunkIterator ` which inherits from `GenericDataChunkIterator` [here](https://github.com/catalystneuro/nwb-conversion-tools/blob/main/nwb_conversion_tools/utils/genericdatachunkiterator.py). That you have a built-in progress bar right @CodyCBakerPhD ?

Reply:
Not really sure what code change this selection is referring to, but for Saksham's comments:

there was a reason behind wrapping the DataChunkIterator due, I recall, to get the `tqdm` progress bar to work correctly as it wrote the actual chunks (in the HDF5 Dataset sense) to the NWBFile (line https://github.com/catalystneuro/nwb-conversion-tools/blob/main/nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py#L224 ensures no actual buffering is going on); I agree that if there is an alternative way of doing it without that then it's not needed, but remembering this PR is focused on the context class alone I think a change to the `chunk_data` option can wait for the other PR refactoring the `MovieDataInterface`.

Reply:
> When chunking is disabled then we should not use the DataChunkIterator at all and just wrap the array with HDF5IO to compress. 

This is not correct. "chunking" means different things in these two different contexts.

> but remembering this PR is focused on the context class alone

yes, this change is incorrect, needs to be removed, and is holding up the rest of the PR

Reply:
> This is not correct. "chunking" means different things in these two different contexts.

Yeah, this really should be changed to `buffer_data` as well. Might even want to not make it boolean, and instead apply a size scale similar to `buffer_gb`; if the user wants to load the entire movie, they can just set an arbitrarily (but hopefully not unsafe) buffer size.

Reply:
I have added back the `DataChunkIterator` like it was before this PR. https://github.com/catalystneuro/nwb-conversion-tools/pull/372/files#diff-57b78875c81d904caa745de93f7771de666b821e9d52586cd03a8328e7a74c6bR226